cypress/integration/create-user.spec.js

```diff
@@ -58,91 +56,6 @@ describe("Creation of new user", () => {
     cy.get("h1").should("have.text", "Users")
   })
 
-  it("should allow user to generate a random password", () => {
-    const emailAddress = "bichardemail1@example.com"
-    cy.task("getPasswordResetCode", emailAddress).then((passwordResetCode) => {
-      const newPasswordToken = generateNewPasswordToken(emailAddress, passwordResetCode)
-      cy.visit(`/login/new-password?token=${newPasswordToken}`)
-      cy.get("body").contains(/first time password setup/i)
-      cy.get(".govuk-inset-text").should("not.exist")
-      cy.get("a[class=govuk-link]").click()
-      cy.get(".govuk-inset-text").should("not.be.empty")
-    })
-  })
-
-  it("should not allow the new user to set their password if it contains sensitive information", () => {
```

Comment:
I think sensitive information test should still be checked as that was an important requirement

Reply:
The subject of this test being the new-password view, which has been removed should mean this is not required, as mentioned in the previous comment I  have not created anything new, all I have done is direct the user towards the reset-password view.